server/src/metadata/mod.rs

```diff
@@ -133,13 +133,14 @@ impl MetadataManager {
         }
     }
 
-    /// If they do not already exist, create the tables used for table and model
-    /// table metadata. A "table_metadata" table that can persist tables is
-    /// created, a "model_table_metadata" table that can persist model tables is
-    /// created, and a "model_table_field_columns" table that can save the index
-    /// of field columns in specific model tables is created. If the tables
-    /// exist or were created, return [`Ok`], otherwise return
-    /// [`rusqlite::Error`].
+    /// If they do not already exist, create the tables used for table and model table metadata.
+    /// * The table_metadata table contains the metadata for tables.
+    /// * The model_table_metadata table contains the main metadata for model tables.
+    /// * The model_table_hash_table_name contains a mapping from each tag hash to the name of the
```

Comment:
"model_table" is used as a prefix to be consistent with "model_table_metadata" and "model_table_field_columns", while "hash" is used instead of "tag_hash" to the [programmatically tables](https://github.com/ModelarData/ModelarDB-RS/blob/dev/optimize-storage-engine/server/src/metadata/mod.rs#L518).